Sources/Benchmark/BenchmarkCommand.swift

```diff
@@ -54,7 +69,16 @@ internal struct BenchmarkCommand: ParsableCommand {
             throw ValidationError(debugBuildErrorMessage)
         }
         if iterations != nil && iterations! <= 0 {
-            throw ValidationError(iterationsErrorMessage)
+            throw ValidationError(positiveNumberError(flag: "--iterations", of: "integer"))
+        }
+        if warmupIterations != nil && warmupIterations! <= 0 {
+            throw ValidationError(positiveNumberError(flag: "--warmup-iterations", of: "integer"))
+        }
+        if maxIterations != nil && maxIterations! <= 0 {
+            throw ValidationError(positiveNumberError(flag: "--max-iterations", of: "integer"))
+        }
+        if minTime != nil && minTime! <= 0 {
```

Comment:
I'm okay with this for now, but it might be reasonable to figure out a way to make this easier to express within Swift Argument Parser... Maybe a custom wrapper around `Int` or `Float`? (Anyway, for another time.)

Reply:
I think it's a great suggestion, that's worth exploring in follow-up PRs.